LocalPayment/src/test/java/com/braintreepayments/api/LocalPaymentClientUnitTest.java

```diff
@@ -764,6 +764,42 @@ public void clearActiveBrowserSwitchRequests_forwardsInvocationToBraintreeClient
         verify(braintreeClient).clearActiveBrowserSwitchRequests(activity);
     }
 
+    @Test
+    public void onBrowserSwitchResult_sends_the_correct_value_of_hasUserLocationConsent_to_getClientMetadataId() throws JSONException {
+        BrowserSwitchResult browserSwitchResult = mock(BrowserSwitchResult.class);
+        when(browserSwitchResult.getStatus()).thenReturn(BrowserSwitchStatus.SUCCESS);
+
+        when(browserSwitchResult.getRequestMetadata()).thenReturn(new JSONObject()
+            .put("payment-type", "ideal")
+            .put("merchant-account-id", "local-merchant-account-id"));
+
+        String webUrl = "sample-scheme://local-payment-success?paymentToken=successTokenId";
+        when(browserSwitchResult.getDeepLinkUrl()).thenReturn(Uri.parse(webUrl));
+        BraintreeClient braintreeClient = new MockBraintreeClientBuilder()
+            .configuration(payPalEnabledConfig)
+            .integration("custom")
+            .sessionId("session-id")
+            .build();
+        when(payPalDataCollector.getClientMetadataId(any(Context.class), same(payPalEnabledConfig), anyBoolean())).thenReturn("client-metadata-id");
+
+        LocalPaymentNonce successNonce = LocalPaymentNonce.fromJSON(new JSONObject(Fixtures.PAYMENT_METHODS_LOCAL_PAYMENT_RESPONSE));
+        LocalPaymentResult localPaymentResult = mock(LocalPaymentResult.class);
+        LocalPaymentApi localPaymentApi = new MockLocalPaymentApiBuilder()
+            .tokenizeSuccess(successNonce)
+            .createPaymentMethodSuccess(localPaymentResult)
+            .build();
+
+        LocalPaymentRequest request = getIdealLocalPaymentRequest();
+        LocalPaymentClient sut = new LocalPaymentClient(activity, lifecycle, braintreeClient, payPalDataCollector, localPaymentApi);
+
+        sut.startPayment(request, localPaymentStartCallback);
+
+        sut.setListener(listener);
+        sut.onBrowserSwitchResult(activity, browserSwitchResult);
+
+        verify(payPalDataCollector).getClientMetadataId(any(), eq(payPalEnabledConfig), eq(true));
```

Comment:
```suggestion
        verify(payPalDataCollector).getClientMetadataId(any(), same(payPalEnabledConfig), eq(true));
```
nit: seems more semantically correct to verify that we're forwarding a reference to the `getClientMetadataId()` invocation.

Reply:
Good call! i've updated to use `same()` instead.